sansa-inference-spark/pom.xml

```diff
@@ -230,6 +231,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
+            <version>1.19</version>
```

Comment:
why here and not in SANSA-Parent?

Reply:
ok, Could you please add this version to SANSA-Parent?

Reply:
It is already in SANSA-Parent: 

https://github.com/SANSA-Stack/SANSA-Parent/blob/bccf80a80e5a06095de2ee62a4f3cd496b222870/pom.xml#L124,L128

Can you rebuild to see if the changes are reflected?

Reply:
Yep, seems to work.